tests/tests/util/options.rs

```diff
@@ -85,6 +86,7 @@ pub struct PlayerOptions {
     max_execution_duration: Option<Duration>,
     viewport_dimensions: Option<ViewportDimensions>,
     with_renderer: Option<RenderOptions>,
+    with_audio: Option<bool>,
```

Comment:
This could just be bool I think, the other things are option because they have different behaviour for not being set at all

Reply:
Ah, alright. I thought they were `Option` so `serde` won't require them in the `.toml`. Changed to simply `bool`.